package/opt/hassbian/suites/install_duckdns.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+function duckdns-show-short-info {
+echo -e "Setup for duckdns auto renewal"
+}
+
+function duckdns-show-long-info {
+echo -e "This script adds an cron job to uppdate you duckdns IP address."
+}
+
+function duckdns-show-copyright-info {
+	echo "Original consept by Ludeeus <https://github.com/ludeeus>"
+}
+
+function duckdns-install-package {
+duckdns-show-long-info
+duckdns-show-copyright-info
+
+echo
+echo "Please take a moment to setup autorenewal of duckdns."
+echo "If no choice is made the installation will exit."
+echo
+echo "(if your domain is 'example.duckdns.org' type example)"
+echo -n "Domain: "
+read domain
+if [ ! "$domain" ]; then
+  exit
+fi
+
+echo -n "Token: "
+read token
+echo
+if [ ! "$token" ]; then
+  exit
+fi
+
+echo "Changing back to pi user..."
```

Comment:
It would be preferable if this was run by the `homeassistant` user.

Reply:
Done :)